discord/guild.py

```diff
@@ -1188,6 +1188,7 @@ async def create_text_channel(
         topic: str = MISSING,
         slowmode_delay: int = MISSING,
         nsfw: bool = MISSING,
+        news: bool = MISSING,
```

Comment:
This could just default to `False`
```suggestion
        news: bool = False,
```